src/compat/dbtype_def.h

```diff
@@ -529,7 +529,7 @@ extern "C"
 #define DB_NUMERIC_UNDERFLOW_LIMIT 1e-38
 
 /* The maximum precision that can be specified for a CHAR(n) domain. */
-#define DB_MAX_CHAR_PRECISION DB_MAX_STRING_LENGTH
+#define DB_MAX_CHAR_PRECISION (DB_MAX_STRING_LENGTH/4)
```

Comment:
please add a blank before and after a binary operator
```suggestion
#define DB_MAX_CHAR_PRECISION (DB_MAX_STRING_LENGTH / 4)
```

Reply:
To improve the readability of code, how about using INTL_UTF8_MAX_CHAR_SIZE macro instead of 4?

Reply:
I referred the below definition to keep similar style:

#define DB_MAX_NCHAR_PRECISION (DB_MAX_STRING_LENGTH/2)
/* The maximum precision that can be specified for a NATIONAL CHARACTER VARYING domain. This probably isn't restrictive enough.	
 * We may need to define this functionally as the maximum precision will depend on the size multiplier of the codeset.	 
*/